homie/mqtt/mqtt_base.py

```diff
@@ -17,16 +17,6 @@
 logger = logging.getLogger(__name__)
 
 
-MQTT_SETTINGS = {
```

Comment:
prefer this list of default settings be left in place

Reply:
hi - that's fine but i believe the exact same list is available in "homie_mqtt_client.py" which is the one that - i believe - gives the actual defaults to the Paho MQTT derived class. so I'd say it's more appropriate in the other file. else there appears to be a duplication with no particular rationale. Maybe i am mistaken though... although i am also using a few defaults.

Reply:
found them there, agree, lets delete them here

Reply:
Can I suggest adding an (optional) setting to specify where the CA certificate file is? My experience with the PAHO client on Raspberry Pi is that it doesn't find the system-wide default (/etc/ssl/certs/ca-certificates.crt seems to work for me), and would be needed for anyone using self-signed certs for development/personal projects.